t/058-tcp-socket.t

```diff
@@ -3690,3 +3690,33 @@ received: OK
 close: 1 nil
 --- no_error_log
 [error]
+
+
+
+=== TEST 61: resolver send query failed
```

Comment:
I think the actual behavior relies on the actual system's network configurations. Can we make this test case more portable please?